packages/content/src/requests/items/freeText/FreeTextAcceptResponseItem.ts

```diff
@@ -16,7 +16,7 @@ export class FreeTextAcceptResponseItem extends AcceptResponseItem implements IF
     @validate()
     public freeText: string;
 
-    public static override from(value: IFreeTextAcceptResponseItem | FreeTextAcceptResponseItemJSON): FreeTextAcceptResponseItem {
+    public static override from(value: IFreeTextAcceptResponseItem | Omit<FreeTextAcceptResponseItemJSON, "@type">): FreeTextAcceptResponseItem {
```

Comment:
you should still allow to add `@type`, so `IX | Omit<XJSON, "@type"> | XJSON` would be better here